### DISCUSSION TITLE
[Projects Beta] Can't filter beta projects in issue search field

### DISCUSSION THREAD

**Discussion by vldt-aem** [community/community#12423] [Projects and Issues] [answered]
It seems like you can't search for a beta project in the issues search field. It only searches for projects that aren't in the beta stage. Is there a way to fix this in the meantime while we wait for the beta projects to replace projects?

**Comment by mariorod** [comment, marked as answer]
HI Andreas (@vldt-aem) ... we are currently working on this feature and will post a link to the changelog when is live this month. Thanks for your patience.

**Reply by vldt-aem** [reply]
@mariorod Thanks!
